libr/core/cmd_print.c

```diff
@@ -5130,7 +5130,7 @@ static int cmd_print(void *data, const char *input) {
 					const char *comma = "";
 					const ut8 *buf = core->block;
 					int withref = 0;
-					for (i = 0; i < core->blocksize; i += (base / 4)) {
+					for (i = 0; i < core->blocksize; i += (base / 8)) {
```

Comment:
uhm i think this 4/8 shuold depend on asm.bits, there's an issue to implement asm.databits, which may be used for those cases, but im afraid to not have enough tests to ensure this change is correct

Reply:
so if anal->bits == 64, i increments 16 bytes, and if its 32, goes for 4.. if so what im thinking is that we should probably use a const wordsize = base / 8; and use `wordsize` instead of evaluating base/8 on each iteration. but the change looks good to me

Reply:
can you change this and ill merge? 